docs/developers.rst

```diff
@@ -75,7 +75,8 @@ is set through environment variables:
 
 **Metadata API**
     Set user/pass using the environment variables ``ALGOSEEK_API_USERNAME`` and
-    ``ALGOSEEK_API_PASSWORD``.
+    ``ALGOSEEK_API_PASSWORD``. If no credentials are provided, the public
+    credentials are used.
```

Comment:
I would drop this part completely, so it does not confuse a client.

Reply:
There is also a mention in the configuration documentation:

https://github.com/algoseekgit/algoseek-connector/blob/97098fc9a9a9c9fe13e428d131179ef518c8bbf6/docs/tutorials/configuration.rst?plain=1#L97-L102

Should I remove it also?

Reply:
Done. Metadata API user/pass is still editable from the Configuration class, but I think that it will be better to remove that once API authorization is removed completely.

Reply:
Agree